trunk/src/app/srs_app_rtc_conn.cpp

```diff
@@ -3131,9 +3131,14 @@ srs_error_t SrsRtcConnection::generate_publish_local_sdp(SrsRequest* req, SrsSdp
     for (int i = 0;  i < (int)stream_desc->video_track_descs_.size(); ++i) {
         SrsRtcTrackDescription* video_track = stream_desc->video_track_descs_.at(i);
 
+        SrsVideoPayload* payload = (SrsVideoPayload*)video_track->media_;
+        if (payload == NULL){
```

Comment:
Should be replaced by:

```cpp
if (!video_track->media_) break;
```

?

Reply:
Thanks for your review! 
Is there any coding style guide? Someone likes me may need it.